code/modules/mob/living/carbon/carbon.dm

```diff
@@ -268,6 +268,10 @@
 
 	if(!item) return
 
+	if(config.atom_content_limit && get_turf(target).get_all_contents_count(list(/obj)) >= config.atom_content_limit)
```

Comment:
Multiple `get_turfs()` of the same object (`get_turf(target)`)
The latter case also double-checks that turf isn't null, this line does not